ui/src/app/tasks/task-execution/task-execution.component.html

```diff
@@ -50,7 +50,7 @@ <h1>Task Execution Details - Execution ID: {{ taskExecution.executionId }}</h1>
     <tr>
       <td>Job Execution Ids</td>
       <td *ngIf="taskExecution.jobExecutionIds.length > 0" id="jobExecutionIds">
-        <a class="btn btn-default" type="button" [routerLink]="'jobs/executions/' + jobExecutionId"
```

Comment:
While the routerLink is fixed now, don't we need the style for the link(s) still?

Reply:
The button render is not really nice in a table, we have to use classic link in that case.